wagtail/test/testapp/models.py

```diff
@@ -1117,6 +1117,11 @@ class CountryCode(models.TextChoices):
     )
     some_date = models.DateField(auto_now=True)
 
+    some_number = models.IntegerField(
+        default=0,
+        blank=True,
+    )
```

Comment:
Nevermind, apparently it could cause issues when the test is run in parallel. (Wonder why I'm not seeing this with other properties when testing `ModelViewSet`...)